src/web/arr/trove/chart.arr

```diff
@@ -1917,6 +1918,42 @@ fun bar-chart-from-list(labels :: P.LoS, values :: P.LoN) -> DataSeries block:
   data-series.make-axis(max-positive-height, max-negative-height)
 end
 
+fun dot-chart-from-list(labels :: P.LoS, values :: P.LoN) -> DataSeries block:
+  doc: ```
+       Consume labels, a list of string, and values, a list of numbers
+       and construct a dot chart
+       ```
+  # Type Checking
+  values.each(check-num)
```

Comment:
Pretty sure this is redundant, given that `P.LoN` does a deep check (https://github.com/brownplt/code.pyret.org/blob/a5adce59c91f544fc1b445bb82191a9092de1bd2/src/web/js/trove/chart-lib.js#L1741)